### PR TITLE
Rename FareProductCode to FareProductId

### PR DESCRIPTION
**Pull request by ue71603**
Should be aligned with type

### CODE REVIEW

File: OJP/OJP_FareSupport.xsd [hunk 1]
```diff
@@ -255,9 +255,9 @@
 			<xs:documentation>[related to FARE PRODUCT in TM and NeTEx] different FARE PRODUCTs that may be available with related information</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="FareProductId" type="FareProductCodeType">
+			<xs:element name="FareProductCode" type="FareProductCodeType">
```
Comment (Aurige) [inline OJP/OJP_FareSupport.xsd:258]: not sure why Code over Id ? in NeTEx the FP has an **Id** and a **PrivateCode**, but not a **Code**

Comment (ue71603) [reply OJP/OJP_FareSupport.xsd:258]: because it is a FareProductCodeType... we can harmonise it the other way.

Comment (ue71603) [reply OJP/OJP_FareSupport.xsd:258]: Transmodel has it as FareProductIdType, so I switch that part.

**Review by AndreasAtSBB — DISMISSED** [review]
Changed text also in the html file: Changed "FareProductCodeType" to "FareProductIdType". OK.

**Review by trurlurl — DISMISSED** [review]

**Review by Aurige — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Pull request merged**